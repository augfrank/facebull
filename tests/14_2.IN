M0   C0   C5  2740
M1   C0   C8  2287
M2   C0   C12  2428
M3   C0   C13  2493
M4   C1   C5  2467
M5   C1   C8  2081
M6   C2   C1  2632
M7   C2   C5  2749
M8   C2   C6  2414
M9   C2   C7  2565
M10   C2   C8  2379
M11   C2   C9  2947
M12   C2   C13  2859
M13   C3   C1  2679
M14   C3   C4  2410
M15   C3   C10  2467
M16   C3   C12  2631
M17   C4   C0  2720
M18   C4   C3  2042
M19   C5   C1  2787
M20   C5   C2  2800
M21   C6   C1  2459
M22   C6   C4  2519
M23   C6   C10  2123
M24   C7   C2  2027
M25   C7   C5  2745
M26   C7   C9  2739
M27   C8   C2  2263
M28   C8   C5  2862
M29   C8   C12  2954
M30   C9   C0  2439
M31   C9   C2  2734
M32   C10   C5  2974
M33   C10   C6  2334
M34   C10   C7  2262
M35   C10   C8  2499
M36   C10   C9  2723
M37   C10   C11  2421
M38   C11   C4  2051
M39   C11   C7  2138
M40   C11   C9  2304
M41   C11   C13  2197
M42   C12   C1  2354
M43   C12   C3  2217
M44   C12   C5  2201
M45   C12   C6  2898
M46   C12   C8  2084
M47   C12   C11  2901
M48   C13   C0  2392
M49   C13   C6  2550
M50   C13   C9  2121
