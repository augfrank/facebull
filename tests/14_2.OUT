33386
3 5 9 16 18 19 23 26 27 30 37 38 44 49
